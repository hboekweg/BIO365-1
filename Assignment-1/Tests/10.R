# This imports your function
source("10.R")

# This invokes your function
print(predictPopulationGrowth("E. coli", 10, 0.20))
print(predictPopulationGrowth("Yersinia pestis", 42847, 0.80))
print(predictPopulationGrowth("Actinobacteria", 9947, 0.0125))
