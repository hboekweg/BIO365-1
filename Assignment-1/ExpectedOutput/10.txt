[1] "After 3 days, there will be 17 E. coli"
[1] "After 3 days, there will be 249884 Yersinia pestis"
[1] "After 3 days, there will be 10325 Actinobacteria"
